bin/generate_unfiltered_sce.R

```diff
@@ -81,13 +95,14 @@ if (opt$feature_dir != ""){
    
   unfiltered_sce <- merge_altexp(unfiltered_sce, feature_sce, opt$feature_name) 
   # add alt experiment features stats
-  altExp(unfiltered_sce, opt$feature_name) <- scater::addPerFeatureQC(altExp(unfiltered_sce, opt$feature_name))
+  altExp(unfiltered_sce, opt$feature_name) <- scuttle::addPerFeatureQCMetrics(altExp(unfiltered_sce, opt$feature_name))
 }
 
 # add per cell and per gene statistics to colData and rowData
 unfiltered_sce <- unfiltered_sce |>
   add_cell_mito_qc(mito = mito_genes) |>
-  scater::addPerFeatureQC()
+  add_gene_symbols(gene_info = gtf) |>
```

Comment:
```suggestion
 # add gene symbols to rowData
  add_gene_symbols(gene_info = gtf) |>
```
Just adding in a small clarifying comment here to remind us (mostly for myself) where the gene symbols are going and point out when they are being added.

Reply:
I'll do it, but I tried to make the function name clear!